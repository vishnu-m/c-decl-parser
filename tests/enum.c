enum colours{Red, Black, Yellow};
enum week{Mon=99, Tue, Wed, Thur, Fri, Sat, Sun};
