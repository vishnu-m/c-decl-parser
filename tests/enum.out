EnumDecl	"colours"		Parent = "/home/vishnu/Desktop/c-decl-parser/tests/enum.c"
EnumConstantDecl	"Red"		Parent = "colours"
EnumConstantDecl	"Black"		Parent = "colours"
EnumConstantDecl	"Yellow"		Parent = "colours"
EnumDecl	"week"		Parent = "/home/vishnu/Desktop/c-decl-parser/tests/enum.c"
EnumConstantDecl	"Mon"		Parent = "week"
IntegerLiteral		 value = "99" 		Parent = "Mon"
EnumConstantDecl	"Tue"		Parent = "week"
EnumConstantDecl	"Wed"		Parent = "week"
EnumConstantDecl	"Thur"		Parent = "week"
EnumConstantDecl	"Fri"		Parent = "week"
EnumConstantDecl	"Sat"		Parent = "week"
EnumConstantDecl	"Sun"		Parent = "week"
